main/algorithm.py

```diff
@@ -5,9 +5,8 @@
 from sys import platform
 from zipfile import ZipFile
 
-from django.http import HttpResponse
-
 import env_consts as ec
+from django.http import HttpResponse
```

Comment:
According to PEP8, importing of (or from) third-party libraries should be done secondly. https://peps.python.org/pep-0008/#imports

Suggestion: rollback the change

Reply:
I am not sure that after such change linter tests will pass

Reply:
I am not sure we should use a tool that goes against the principles of the language. Ok, if there is no way to overcome it, I will mark this one as resolved.